3

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
Is this really a file called `3`? Or is there some kind of problem with this diff?

Reply:
removed!